neo4j/internal/pool/pool.go

```diff
@@ -414,3 +418,21 @@ func (p *Pool) Return(c db.Connection) {
 		p.log.Warnf(log.Pool, p.logId, "Server %s not found", serverName)
 	}
 }
+
```

Comment:
Both of these are accessing structures written to from other threads, will not work.